pkg/chains/chains.go

```diff
@@ -29,7 +17,9 @@ var (
 		Consensus:   Consensus_tendermint,
 		IsExternal:  false,
 	}
-	EthChain = Chain{
+
+	// Ethereum is Ethereum mainnet
+	Ethereum = Chain{
```

Comment:
NIT: going by the naming, this should be EthereumMainnet . Same for Polygon .

Reply:
There reason for this one is that Ethereum and Polygon use names for their testnet, so `Ethereum` already reflect a mainnet since the testnet is Sepolia
Note: We could also use `Athens3` for ZetaChain but it is a bit ugly and then would become inconsistent with localnet